src/components/menu/__tests__/__snapshots__/menu-popover.test.tsx.snap

```diff
@@ -12,6 +12,7 @@ exports[`Menu Popover component should render and close Popover based on any ele
     </button>
     <mock-styled.div
       data-open="true"
+      style="top: 3px;"
```

Comment:
**question:** I can't see where you've gotten this 3px value from in the Figma files?

Reply:
actually I just manually calculating it, as you can see in below screenshot without additional `+3` it look like this
![image](https://github.com/user-attachments/assets/ff942510-3c44-4c43-86d4-409d8f5b96bf)
![image](https://github.com/user-attachments/assets/eb06c139-ffcf-4c2f-b297-6a09aa6e8724)

with `+3`
![Uploading image.png…]()

note: there is no example for the `Menu` usage with `button` in the design as far I've seen (except nav dropdown button which would need more gap) 🤔

Reply:
Okay, I thought `3px` was a strange number. Can you please ask design what the default offset should be? 🙏 It's not our decision to make, unless they explicitly tell us we can make the decision.

For reference, `3px` seems strange to me because it's not one of the spacing design tokens. If design were to specify this offset, I would expect them to use one of their `--space-*` tokens, likely `--space-half` (2px) or `--space-1` (4px), but again, this kind of detail is for Design to specify, not us.

Reply:
yes, I guess since the current example only for app-bar (and we will use it for app-bar initially) can we adjust the gap to match the app bar example? , there is no offset prop in the spec but we can calculate it using same app-bar height value which is `size-10` or default to `0` instead, so just need to custom y offset with props?

![image](https://github.com/user-attachments/assets/986719bf-243a-43aa-8798-081f7b0e58cd)